Sources/Sentry/Public/SentryOptions.h

```diff
@@ -187,6 +187,13 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableNetworkTracking;
 
+/**
+ * THIS FLAG IS EXPERIMENTAL.
+ * When enabled, the SDK tracks performance for IO
+ * reads and writes if auto performance tracking is enabled. The default is <code>NO</code>
+ */
+@property (nonatomic, assign) BOOL enableIOTracking;
```

Comment:
`m`: Shouldn't it be `enableFileIOTracking`? IO could be any input & output.
```suggestion
@property (nonatomic, assign) BOOL enableFileIOTracking;
```